### PR TITLE
Resolve GraphQL dependency issue

### PR DESCRIPTION
**Pull request by ericmand**
To address issues #3 and #4 per @benjie's comment in #4

### CODE REVIEW

**Comment by ericmand** [comment]
I moved to peerDependancies and added a caret. I haven't, but just noticed and probably should, remove the "resolutions" section, as that seems like it might be problematic to me.

File: package.json [hunk 1]
```diff
@@ -35,10 +35,12 @@
   "dependencies": {
     "apollo-link": "^1.2.3",
     "gatsby-source-graphql": "^2.0.5",
-    "graphql": "0.13.x",
     "pg": "^7.6.0",
     "postgraphile": "^4.1.0-rc.0"
   },
+  "peerDependencies": {
+    "graphql": "^0.13.x"
```
Comment (benjie) [inline package.json]: We should add 14 to the list too, I think, e.g.:

```suggestion
    "graphql": "^0.13.x | ^14.0.0"
```

Comment (benjie) [reply package.json]: What version range of GraphQL does Gatsby support?

Comment (ericmand) [reply package.json]: Gatsby's package.json has "graphql": "^14.1.1" in its dependancies. Should we just use that?

**Comment by ericmand** [comment]
Remind me, anything else you need from me? Do you merge or do I?

**Comment by benjie** [comment]
Sorry; plan is to merge and release when I get a moment. Leaving it open to remind myself. Hopefully before the weekend 👍

**Comment by benjie** [comment]
0.1.3 released with this, and quickly replaced with 0.1.4 which basically just mimicks https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-graphql/package.json

**Comment by benjie** [comment]
I also upgraded PostGraphile while I was at it

**Pull request merged**